### ISSUE TITLE
Request: Expose ffmpeg compression backend

### ISSUE THREAD

**Issue by RollingStar** [john32b/cdcrush.net#6]
This might be better than a a few-size-fits-all approach. You're already using ffmpeg to convert, so it would be cool if the program just generated a text string that got passed to ffmpeg. It would expose commands like [this](https://github.com/johndimi/cdcrush.net/blob/280d9574e6dd29027ee44f15e4f8af5a7f7a60e7/cdcrush/lib/app/FFmpeg.cs#L165) and allow for user editing.

Some video compression tools already use this approach. It is great for power users, because they can customize the command as much as they want. It also gives the tool more forward-compatibility.

It would be even better if the tool loaded options from a config file for the ffmpeg path. Something like:

```
[option-flac]
command: ffmpeg -i %1 %1.flac
[option-ogg]
command: ffmpeg -i %1 %1.ogg
```

Then I could add Opus recipes at different bitrates, for example. :)

**Comment by john32b** [comment]
Yes, I'll look into a way to achieve something like this, it seems useful 👍 
Perhaps as an advanced setting/option, since I don't want to take away from the usability of the program!
